deploy/memcache-account.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
I think you forgot to remove this

Reply:
Yep missed that after removing the memcached service account.. Removed the file now.